plugins/modules/discovery_workflow_manager.py

```diff
@@ -49,9 +49,10 @@
         elements: str
         required: true
       discovery_type:
-        description: Type of discovery (SINGLE/RANGE/MULTI RANGE/CDP/LLDP)
+        description: Type of discovery (SINGLE/RANGE/MULTI RANGE/CDP/LLDP/CIDR)
         type: str
         required: true
+        choices: [ 'SINGLE', 'RANGE', 'MULTI RANGE', 'CDP', 'LLDP', 'CIDR']
```

Comment:
Can we explain these choices what is what?

Reply:
Added